MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,961 @@
+---
+layout: post
+title: Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI SfDataForm
```

Comment:
Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI DataForm (SfDataForm)

Reply:
Changed title